### DISCUSSION TITLE
Set initial index

### DISCUSSION THREAD

**Discussion by ccboyce** [prabhuignoto/react-chrono#154] [Q&A] [answered]
Is it possible to set an initial index on load, rather than defaulting to the first element in the array?
Thank you.

**Comment by prabhuignoto** [comment, marked as answer]
you can use  `focusActiveItemOnLoad` in combination with `activeItemIndex` to automatically set a specific index on load

refer the props section
https://github.com/prabhuignoto/react-chrono#props

example:

```
<Chrono
  items={items}
  mode="VERTICAL"
  focusActiveItemOnLoad
  activeItemIndex={2}
/>
```
